packages/main/src/TabContainer.ts

```diff
@@ -100,6 +99,20 @@ type TabContainerMoveEventDetail = {
 	}
 }
 
+/**
+ * Interface for components that may be slotted inside `ui5-tabcontainer` as items
+ *
+ * **Note:** Use directly `ui5-tab` or `ui5-tab-seprator`. Implementing the interface does not guarantee that the class can work as a tab.
+ * @public
+ */
+interface ITab extends UI5Element {
+	isSeparator: boolean;
+	receiveStripInfo: (arg0: TabContainerStripInfo) => void;
+	receiveOverflowInfo: (arg0: TabContainerOverflowInfo) => void;
+	getDomRefInStrip: () => HTMLElement | undefined;
+	items?: Array<ITab>;
```

Comment:
I think "items" shouldn't be here. It is never implemented by TabSeparator and is not needed for a tab item?

Reply:
It is true that is not needed by the TabSeparator, but it seems generic enough to be put in the interface.
This way we avoid multiple `as Tab` casts in the TabContainer, and can work directly with the interface.